chaco/api.py

```diff
@@ -325,39 +325,45 @@
 from .tooltip import ToolTip
 from .data_label import DataLabel
 from .lasso_overlay import LassoOverlay
-from .color_bar import ColorBar
+from .plots.color_bar import ColorBar
 from .text_box_overlay import TextBoxOverlay
 from .scatter_inspector_overlay import ScatterInspectorOverlay
 from .colormapped_selection_overlay import ColormappedSelectionOverlay
 
 # Renderers
-from .barplot import BarPlot
 from .base_1d_plot import Base1DPlot
 from .base_2d_plot import Base2DPlot
 from .base_xy_plot import BaseXYPlot
-from .scatterplot import ScatterPlot, render_markers
-from .image_plot import ImagePlot
-from .cmap_image_plot import CMapImagePlot
-from .contour_line_plot import ContourLinePlot
-from .contour_poly_plot import ContourPolyPlot
-from .lineplot import LinePlot
-from .colormapped_scatterplot import ColormappedScatterPlot
-from .polygon_plot import PolygonPlot
-from .errorbar_plot import ErrorBarPlot
-from .filled_line_plot import FilledLinePlot
-from .quiverplot import QuiverPlot
-from .candle_plot import CandlePlot
-from .multi_line_plot import MultiLinePlot
-from .jitterplot import JitterPlot
-from .horizon_plot import HorizonPlot
-from .scatterplot_1d import ScatterPlot1D
-from .line_scatterplot_1d import LineScatterPlot1D
-from .text_plot_1d import TextPlot1D
-from .segment_plot import SegmentPlot
-from .text_plot import TextPlot
-from .polar_line_renderer import PolarLineRenderer
+
+from chaco.plots.api import (
+    BarPlot,
+    CandlePlot,
+    CMapImagePlot,
+    ColormappedScatterPlot,
```

Comment:
```suggestion
    ColormappedScatterPlot,
    ColormappedScatterPlotView,
    ColormappedSegmentPlot,
```

Reply:
can you also include this - and other missing imports i suggested in this module - to the module docstring?